sys-apps/man-db/man-db-2.7.6.1-r3.ebuild

```diff
@@ -105,4 +113,9 @@ pkg_postinst() {
 		einfo "Rebuilding man-db from scratch with new database format!"
 		mandb --quiet --create
 	fi
+
+	if use systemd; then
```

Comment:
I respect your preferences but when Gentoo added support for systemd we explicit voted against the use of a systemd USE flag to control such things. So at the moment, a contribution adding a systemd USE flag for things like that has no chance of getting merged.